.github/workflows/update-search.yml

```
@@ -35,21 +35,37 @@ jobs:
+            # Fetch and pull latest changes before committing
+            git pull origin main --rebase
+            
             # Commit changes
             git commit -m "Update search database from comphy-search repository [skip-deploy]"
             
-            # Rebase onto latest origin/main to avoid conflicts
-            git rebase origin/main
+            # Push with retry logic
+            MAX_RETRIES=3
+            RETRY_COUNT=0
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)